src/Tribe/Views/V2/Assets.php

```diff
@@ -423,11 +453,6 @@ public function register() {
 	 * @return void
 	 */
 	public function disable_v1() {
-		// Dont disable V1 on Single Event page
-		if ( tribe( Template_Bootstrap::class )->is_single_event() ) {
-			return;
-		}
```

Comment:
Just to confirm our intention is to remove V1 styles completely correct?

Reply:
@bordoni yes